lib/driver.js

```diff
@@ -401,6 +402,30 @@ class XCUITestDriver extends BaseDriver {
     }
   }
 
+  async uninstallWDAIfRunningBundleIdIsDifferent () {
```

Comment:
I think it makes sense to move this method to WebDriverAgent class, since it anyway mostly uses `this.wda`